aws-lambda-java-serialization/src/main/java/com/amazonaws/services/lambda/serialization/JacksonPojoSerializer.java

```diff
@@ -0,0 +1,222 @@
+                DeserializationFeature.FAIL_ON_INVALID_SUBTYPE,			    //this is default as of 2.12.3
+                DeserializationFeature.FAIL_ON_UNRESOLVED_OBJECT_IDS,		//this is default as of 2.12.3
+                DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_AS_NULL,
+                DeserializationFeature.UNWRAP_SINGLE_VALUE_ARRAYS,
+                DeserializationFeature.WRAP_EXCEPTIONS				        //this is default as of 2.12.3
+        );
+        dcfg = dcfg.withoutFeatures(
+                DeserializationFeature.FAIL_ON_IGNORED_PROPERTIES,		    //this is default as of 2.12.3
+                DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES,		    //this is default as of 2.12.3
+                DeserializationFeature.FAIL_ON_NUMBERS_FOR_ENUMS,		    //this is default as of 2.12.3
+                DeserializationFeature.FAIL_ON_READING_DUP_TREE_KEY,		//this is default as of 2.12.3
+                DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES
+        );
+        mapper.setConfig(dcfg);
+        mapper.setSerializationInclusion(Include.NON_NULL);                 //NON_EMPTY?
+
+        mapper.enable(MapperFeature.ALLOW_FINAL_FIELDS_AS_MUTATORS);		//this is default as of 2.12.3
+        mapper.enable(MapperFeature.AUTO_DETECT_FIELDS);			        //this is default as of 2.12.3
+        mapper.enable(MapperFeature.AUTO_DETECT_GETTERS);			        //this is default as of 2.12.3
+        mapper.enable(MapperFeature.AUTO_DETECT_IS_GETTERS);			    //this is default as of 2.12.3
+        mapper.enable(MapperFeature.AUTO_DETECT_SETTERS);			        //this is default as of 2.12.3
+        mapper.enable(MapperFeature.CAN_OVERRIDE_ACCESS_MODIFIERS);		    //this is default as of 2.12.3
+        mapper.enable(MapperFeature.USE_STD_BEAN_NAMING);
+        mapper.enable(MapperFeature.USE_ANNOTATIONS);				        //this is default as of 2.12.3
+
+        mapper.disable(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES);	//this is default as of 2.12.3
+        mapper.disable(MapperFeature.AUTO_DETECT_CREATORS);
+        mapper.disable(MapperFeature.INFER_PROPERTY_MUTATORS);
+        mapper.disable(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY);		//this is default as of 2.12.3
+        mapper.disable(MapperFeature.USE_GETTERS_AS_SETTERS);
+        mapper.disable(MapperFeature.USE_WRAPPER_NAME_AS_PROPERTY_NAME);	//this is default as of 2.12.3
+        mapper.disable(MapperFeature.USE_STATIC_TYPING);			        //this is default as of 2.12.3
+        mapper.disable(MapperFeature.REQUIRE_SETTERS_FOR_GETTERS);		    //this is default as of 2.12.3
+
+        SimpleModule module = new SimpleModule();
+        module.addDeserializer(Void.class, new VoidDeserializer());
+        mapper.registerModule(module);
+
+        JavaTimeModule javaTimeModule = new JavaTimeModule();
+        mapper.registerModule(javaTimeModule);
+
+        return mapper;
+    }
+
+    public static final class VoidDeserializer extends JsonDeserializer<Void> {
```

Comment:
What is this for ?

Reply:
In order to serialize `Void` as not `null`.

Reply:
Can you give me an example ?

Reply:
Sure, a specific test for it was created https://github.com/aws/aws-lambda-java-libs/blob/c1685c79742aae77609d9dd1beb38f214a0664b1/aws-lambda-java-serialization/src/test/java/com/amazonaws/services/lambda/serialization/JacksonPojoSerializerTest.java

Reply:
I'm still trying to understand why you need the `Void` type. Is there any real-world example?